tracing/src/main/java/com/palantir/tracing/CloseableTracer.java

```diff
@@ -35,6 +36,13 @@ public static CloseableTracer startSpan(String operation) {
         return startSpan(operation, SpanType.LOCAL);
     }
 
+    /**
+     * Opens a new {@link SpanType#LOCAL LOCAL} span for this thread's call trace, labeled with the provided operation.
+     */
+    public static CloseableTracer startSpan(String operation, Map<String, String> metadata) {
```

Comment:
It depends. Most uses of this class are for `LOCAL` spans, so I think it's helpful to make it obvious that's the simplest choice without necessarily understanding what the span types mean. Realistically there's not much reason for CloseableTracer to support other span types at all -- server incoming requires additional data that CloseableTracer doesn't take (is_sampled, traceId, parentSpanId, etc). I suppose it could work with client outgoing spans now that we have the Tracer.maybeGetMetadata method, but that's almost exclusively used in code we own.